vlasovsolver/cuda_acc_map_kernel.cuh

```diff
@@ -49,6 +49,6 @@ extern Realf* acceleration_1_wrapper
 );
 
 #define DIMS 1
-#define BLOCKS 8
+#define BLOCKS 108
 #define THREADS 32
 #define CUDASIZE 1
```

Comment:
Shouldn't these be Makefile-architecture-defines?

Reply:
Indeed, yes. THREADS is already changed to VECL in the CUDA_REALF version, but blocks depends on the architecture. I'll add something about that.